staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/union_content.go

```diff
@@ -48,7 +48,9 @@ func (c unionCAContent) Name() string {
 func (c unionCAContent) CurrentCABundleContent() []byte {
 	caBundles := [][]byte{}
 	for _, curr := range c {
-		caBundles = append(caBundles, curr.CurrentCABundleContent())
+		if currCABytes := curr.CurrentCABundleContent(); len(currCABytes) > 0 {
+			caBundles = append(caBundles, []byte(strings.TrimSpace(string(currCABytes))))
```

Comment:
why the trim space?

Reply:
is a trailing newline required to parse PEM blocks correctly?

Reply:
https://www.google.com/search?q=pem+trailing+newline makes me nervous about trimming trailing newlines... what was the reason for this?